runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -9806,6 +9806,36 @@ static TR::Register *inlineSinglePrecisionFPTrg1Src2(TR::Node *node, TR::InstOpC
    return targetRegister;
    }
 
+static TR::Register *inlineFPTrg1Src3(TR::Node *node, TR::InstOpCode::Mnemonic op, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT_FATAL(node->getNumChildren() == 3, "There should be exactly 3 children in the node of inlineFPTrg1Src3");
```

Comment:
Perhaps you could output the node address and the number of children actually found as part of the message to make this a bit more helpful.

Reply:
Fixed in 05e6a71